src/converse-controlbox.js

```diff
@@ -103,6 +104,12 @@ converse.plugins.add('converse-controlbox', {
                     }
                     view.close();
                 });
+
+                const servive_administration_chat = _converse.chatboxviews.el.querySelector('#sendAdminCommandId');
```

Comment:
There's a typo here, `servive` needs to be `service`. Also, this variable name is very long, everywhere where you use `administration` you can shorten it to `admin`.